projects/angular/src/data/datagrid/datagrid-virtual-scroll.directive.ts

```diff
@@ -109,23 +109,14 @@ export class ClrDatagridVirtualScrollDirective<T> implements AfterViewInit, DoCh
     private readonly directionality: Directionality,
     private readonly scrollDispatcher: ScrollDispatcher,
     private readonly viewportRuler: ViewportRuler,
-    private readonly datagrid: ClrDatagrid,
```

Comment:
Would `@Inject(forwardRef(() = ClrDaragrid)` work here? I think that would be much better than a public `prepareDatagrid` method.

Reply:
I'm glad it worked. Thanks!